packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -347,6 +366,25 @@ abstract class TableBase extends Resource implements ITable {
     });
   }
 
+  /**
+   * Adds an IAM policy statement associated with this table's stream to an
+   * IAM principal's policy.
+   * @param grantee The principal (no-op if undefined)
+   * @param actions The set of actions to allow (i.e. "dynamodb:DescribeStream", "dynamodb:GetRecords", ...)
+   */
+  public grantStream(grantee: iam.IGrantable, ...actions: string[]): iam.Grant {
```

Comment:
Enforce at least one action, no?

```suggestion
  public grantStream(grantee: iam.IGrantable, action: string, ...moreActions: string[]): iam.Grant {
```

Reply:
I've literally just moved the code from `Table` to `TableBase` (due to the requirements of the method being satisfied earlier). I didn't want to change the semantics, but it definitely sounds like a thing that should be checked.